english/explore-outside/talk-child/silent-approach/silent-approach.md

```diff
@@ -5,4 +5,6 @@ that is around you.
 
 You have now reached the child. What do you do?
 
-[Talk to the child anyway](../talk-more/talk-more.md)
\ No newline at end of file
+[Talk to the child anyway](../talk-more/talk-more.md)
+
+[Run away!](../run-away/run-away.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.